docs/UserGuide.adoc

```diff
@@ -44,7 +44,7 @@ Note the following symbols and formatting used in this document:
 [%autowidth]
 |=====
 ^|*Symbol* |*Meaning*
-^|kbd:[Enter] | The enter button on the keyboard.
+^|Enter | The enter button on the keyboard.
```

Comment:
why? it gives a unique visual tho

Reply:
The visual doesn't work when exporting to pdf.